python/graphstorm/gconstruct/config_conversion/gsprocessing_converter.py

```diff
@@ -0,0 +1,329 @@
+                    # Fill in missing values if needed
+                    for entry in ["train", "valid", "test", "column"]:
+                        entry_val = label_custom_split_filenames.get(entry, None)
+                        if entry_val:
+                            if isinstance(entry_val, str):
+                                label_custom_split_filenames[entry] = [entry_val]
+                            else:
+                                assert isinstance(
+                                    entry_val, list
+                                ), "Custom split filenames should be a string or a list of strings"
+                        else:
+                            label_custom_split_filenames[entry] = []
+                    assert len(label_custom_split_filenames["column"]) <= 2, (
+                        "Custom split filenames should have one column for node labels, "
+                        "and two columns for edges labels exactly"
+                    )
+
+                    label_dict["custom_split_filenames"] = {
+                        "train": label_custom_split_filenames["train"],
+                        "valid": label_custom_split_filenames["valid"],
+                        "test": label_custom_split_filenames["test"],
+                        "column": label_custom_split_filenames["column"],
+                    }
+                if "separator" in label:
+                    label_sep = label["separator"]
+                    label_dict["separator"] = label_sep
+                if "mask_field_names" in label:
+                    label_dict["mask_field_names"] = label["mask_field_names"]
+
+                labels_list.append(label_dict)
+            except KeyError as exc:
+                raise KeyError(
+                    f"A required key was missing from label input {label}"
+                ) from exc
+        return labels_list
+
+    @staticmethod
+    def _convert_feature(feats: list[Mapping[str, Any]]) -> list[dict]:
+        """Convert the feature config
+        Parameters
+        ----------
+        feats: list[Mapping[str, Any]]
+            The feature information in the GSProcessing format
+
+        Returns
+        -------
+        list[dict]
+            The feature information in the GConstruct format
+        """
+        gconstruct_feats_list = []
+        if feats in [[], [{}]]:
+            return []
+        for gsp_feat_dict in feats:
+            gconstruct_feat_dict = {}
+            gconstruct_feat_dict["feature_col"] = gsp_feat_dict["column"]
+            if "name" in gsp_feat_dict:
+                gconstruct_feat_dict["feature_name"] = gsp_feat_dict["name"]
+
+            gconstruct_transformation_dict: dict[str, Any] = {}
+            if "transformation" in gsp_feat_dict:
+                gsp_transformation_dict = gsp_feat_dict["transformation"]
+
+                ### START TRANSFORMATION ALTERNATIVES ###
+                if gsp_transformation_dict["name"] == "numerical":
```

Comment:
We have support for string-vectors+truncating in no-op Gconstruct now. Can we ensure to include that?